packages/cli/src/util/link/repo.ts

```
@@ -275,31 +274,23 @@ export async function ensureRepoLink(
       output.spinner(`Creating new Project: ${orgAndName}`);
       delete selection.newProject;
       if (!selection.rootDirectory) delete selection.rootDirectory;
-      try {
-        const project = (selected[i] = await createProject(client, {
-          ...selection,
-          framework: selection.framework.slug,
-        }));
-        await connectGitProvider(
-          client,
-          org,
-          project.id,
-          parsedRepoUrl.provider,
-          `${parsedRepoUrl.org}/${parsedRepoUrl.repo}`
-        );
-        output.log(
-          `Created new Project: ${output.link(
-            orgAndName,
-            `https://vercel.com/${orgAndName}`,
-            { fallback: false }
-          )}`
-        );
-      } catch (err) {
```

This try/catch was the main issue that this PR is fixing. Any error except for "too_many_projects" was being silently ignored.